recipe_sha: 89e676df1fb3317c809e40d951a4384a5421afdc251200433ea8a1ed2d0d5a26  # The hash of the recipe's meta.yaml at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |-
  2024-08-27T18:45:44 noarch Writing index HTML
  /opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py:501: UserWarning: RECIPE_PATH received is a file (/opt/recipe/meta.yaml).
  It should be a path to a folder.
  Forcing conda-build to use the recipe file.
    warnings.warn(
  WARNING: No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.22
  Adding in variants from internal_defaults
  Adding in variants from /opt/host-conda-bld/conda_build_config_0_-e_conda_build_config.yaml
  Adding in variants from /opt/host-conda-bld/conda_build_config_1_-e_bioconda_utils-conda_build_config.yaml
  Attempting to finalize metadata for ucsc-chainantirepeat
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  BUILD START: ['ucsc-chainantirepeat-469-h9b8f530_0.tar.bz2']
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Solving environment (_h_env): ...working... done

  ## Package Plan ##

    environment location: /opt/conda/conda-bld/ucsc-chainantirepeat_1724784345766/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl


  The following NEW packages will be INSTALLED:

      _libgcc_mutex:     0.1-conda_forge      conda-forge
      _openmp_mutex:     4.5-2_gnu            conda-forge
      ca-certificates:   2024.7.4-hbcca054_0  conda-forge
      libgcc-ng:         14.1.0-h77fa898_0    conda-forge
      libgomp:           14.1.0-h77fa898_0    conda-forge
      libopenssl-static: 3.3.1-hb9d3cd8_3     conda-forge
      libpng:            1.6.43-h2797004_0    conda-forge
      libstdcxx-ng:      14.1.0-hc0a3c3a_0    conda-forge
      libuuid:           2.38.1-h0b41bf4_0    conda-forge
      libzlib:           1.2.13-h4ab18f5_6    conda-forge
      mysql-connector-c: 6.1.11-h659d440_1008 conda-forge
      openssl:           3.3.1-hb9d3cd8_3     conda-forge
      zlib:              1.2.13-h4ab18f5_6    conda-forge

  Preparing transaction: ...working... done
  Verifying transaction: ...working... done
  Executing transaction: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Solving environment (_build_env): ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Solving environment (_test_env): ...working... done

  ## Package Plan ##

    environment location: /opt/conda/conda-bld/ucsc-chainantirepeat_1724784345766/_build_env


  The following NEW packages will be INSTALLED:

      _libgcc_mutex:                        0.1-conda_forge     conda-forge
      _openmp_mutex:                        4.5-2_gnu           conda-forge
      _sysroot_linux-64_curr_repodata_hack: 3-h69a702a_16       conda-forge
      binutils_impl_linux-64:               2.40-ha1999f0_7     conda-forge
      binutils_linux-64:                    2.40-hb3c18ed_0     conda-forge
      gcc_impl_linux-64:                    12.4.0-hb2e57f8_0   conda-forge
      gcc_linux-64:                         12.4.0-h6b7512a_0   conda-forge
      kernel-headers_linux-64:              3.10.0-h4a8ded7_16  conda-forge
      ld_impl_linux-64:                     2.40-hf3520f5_7     conda-forge
      libgcc-devel_linux-64:                12.4.0-ha4f9413_100 conda-forge
      libgcc-ng:                            14.1.0-h77fa898_0   conda-forge
      libgomp:                              14.1.0-h77fa898_0   conda-forge
      libsanitizer:                         12.4.0-h46f95d5_0   conda-forge
      libstdcxx-ng:                         14.1.0-hc0a3c3a_0   conda-forge
      make:                                 4.4.1-hb9d3cd8_0    conda-forge
      sysroot_linux-64:                     2.17-h4a8ded7_16    conda-forge
      tzdata:                               2024a-h8827d51_1    conda-forge

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-build", line 11, in <module>
      sys.exit(execute())
    File "/opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 590, in execute
      api.build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 250, in build
      return build_tree(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3638, in build_tree
      packages_from_this = build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2409, in build
      create_build_envs(top_level_pkg, notest)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2253, in create_build_envs
      environ.create_env(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/environ.py", line 1049, in create_env
      _execute_actions(prefix, precs)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/environ.py", line 1332, in _execute_actions
      progressive_fetch_extract.execute()
    File "/opt/conda/lib/python3.10/site-packages/conda/core/package_cache_data.py", line 891, in execute
      raise CondaMultiError(not_cancelled)
  conda.CondaMultiError: Error with archive /opt/conda/pkgs/sysroot_linux-64-2.17-h4a8ded7_16.conda.  You probably need to delete and re-download or re-create this file.  Message was:

  failed with error: [Errno 28] No space left on device
  Failed to write to /opt/conda/pkgs/gcc_impl_linux-64-12.4.0-hb2e57f8_0.conda.partial
    errno: 28
  Failed to write to /opt/conda/pkgs/gcc_impl_linux-64-12.4.0-hb2e57f8_0.conda.partial
    errno: 28
# Last 100 lines of the build log.
